src/components/Dropdown/Dropdown.tsx

```diff
@@ -287,7 +302,16 @@ export default class Dropdown extends AutoControlledComponent<
                         variables,
                       )
                     : this.renderTriggerButton(styles, getToggleButtonProps)}
-                  {toggleButton && this.renderToggleButton(getToggleButtonProps, classes, isOpen)}
+                  {/* TODO: Make `indicator` a fully working shorthand. */}
+                  {DropdownIndicator.create(indicator && '', {
```

Comment:
I don't understand what is the connection between the loading and indicator aspects of the Dropdown... We can replace the toggleButton with indicator (totally support this idea, as it would be consistent with the other components), and the indicator definitely can be shorthand for the Indicator component. The loading is completely separated concept that again can be implemented with a shorthand for the Loading component, and the Dropdown can handle when it should show it..

Reply:
Resolved with latest changes 👍